app/Notifications/ArticleSubmitted.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Notifications;
+
+use App\Models\Article;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Notifications\Notification;
+use NotificationChannels\Telegram\TelegramMessage;
+
+class ArticleSubmitted extends Notification implements ShouldQueue
+{
+    use Queueable;
+
+    private Article $article;
+
+    public function __construct(Article $article)
+    {
+        $this->article = $article;
+    }
+
+    public function via($notifiable)
+    {
+        return ['telegram'];
+    }
+
+    public function toTelegram($notifiable)
+    {
+        $url = route('articles.show', $this->article->slug());
+
+        return TelegramMessage::create()
+            ->to(env('TELEGRAM_LARAVELIO_CHANNEL'))
```

Comment:
I think we could move the telegram specific credentials to the `services.php` config file similar to how we have done with Twitter.